test/java/seedu/addressbook/logic/CommandAssertions.java

```diff
@@ -65,9 +91,10 @@ public static void assertCommandBehavior(String inputCommand,
     }
 
     /**
-     * Executes the command and confirms that the result message is correct.
+     * Executes the command and confirms that the result messages are correct.
```

Comment:
Technically, this has only one message

Reply:
Well, but the output console message should be empty